server/src/api/resources/subscribe/subscribe.controller.js

```diff
@@ -0,0 +1,80 @@
+import * as userController from "../user/user.controller"
+
+export const getSubscription = async (req, res, stripe) => {
+  const { subscribeId } = req.params
+  const subscription = await stripe.subscriptions.retrieve(subscribeId)
+  if (subscription) {
+    return res.status(200).send(subscription)
+  } else {
+    return res.status(500).send("Stripe error: Cannot retrieve subscription.")
+  }
+}
+
+export const subscribeToFreePlan = async (req, res, stripe) => {
+  const { source } = req.body
+  const customer = await stripe.customers.create({ source: source.id })
+
+  if (customer) {
+    const updatedRequest = {
+      ...req,
+      body: {
+        subscribed: true,
+        subdate: Date.now(),
```

Comment:
Let's change all occurrences of `subdate` to `subDate` to align with changes made in my PR